descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,124 @@
+{
+  "id": "${artifactId}-${version}",
+  "name": "Real Time Availability Check Module",
+  "provides": [
+    {
+      "id": "${artifactId}",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/oai/records",
+          "permissionsRequired": ["oai-pmh.records.collection.get"],
+          "modulePermissions": [
+            "inventory-storage.instances.collection.get",
+            "inventory-storage.instances.item.get",
+            "inventory-storage.items.collection.get"
+          ]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/oai/records/{id}",
+          "permissionsRequired": ["oai-pmh.records.record.get"],
```

Comment:
don't forget to update this if you change the permission name (see comment below)